docs/02.API-REFERENCE.md

```diff
@@ -3768,6 +3768,96 @@ jerry_value_to_bigint (const jerry_value_t value);
 - [jerry_value_is_bigint](#jerry_value_is_bigint)
 - [jerry_get_bigint_digits](#jerry_get_bigint_digits)
 
+## jerry_value_to_integer
+
+**Summary**
+
+Call the integer ecma builtin operation on the api value.
+
+**Prototype**
+
+```c
+jerry_value_t
+jerry_value_to_integer (const jerry_value_t value, double *number_p);
```

Comment:
I think this is not a conversion operation, since the result is undefined on success. It is more like a value casting, so `jerry_value_as_integer` could be a better name. Maybe an `jerry_value_as_number` could be added as well for convenience.

Reply:
I cannot agree with that. If you can ensure that the value passed to `jerry_value_as_integer` is an number then this is a `cast`.  This method creates integer number from any JS value so it's definitely a conversion.  I'd keep the `jerry_value_to_integer` (and same for the other ones) name for the function. Also better keep it sync with the abstract operation defined in the standard.